index.js

```diff
@@ -16,10 +17,10 @@ ${title}
 ${favicon}
 ${css}
 ${head}
+${script}
 </head>
 <body>
 ${body}
-${script}
```

Comment:
Then the new possible configurations will be:
- script tag in head + sync
- script tag in body + async (new default)